resources/generate_book.py

```diff
@@ -499,6 +499,18 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
                 multiblock('', '', False, (('   ', '   ', '   '), ('   ', ' R ', '   '), ('   ', ' 0 ', '   ')), {'0': 'AIR', 'R': 'tfc:rock/raw/gabbro'}),
             ),
         )),
+        entry("leather_making","Leather Making", "minecraft:leather", pages=(
+            text("$(thing)Leather$() is a useful item that could be used in the creation of leather armor or used in crafting recipes, but it takes some steps to make it.$(br2)To get started you will first need to obtain hide. This can be done by slaughtering $(l:the_world/wild_animals)animals$().$(br2)Each $(l:the_world/wild_animals)animal$() can drop small, medium or large hide depending on it's size."),
+            item_spotlight("tfc:powder/flux",link_recipe=True,text_contents="Next step is obtaining $(l:mechanics/flux)Flux$().$(br)After you are done obtaining $(l:mechanics/flux)Flux$() you will need to craft a $(l:mechanics/barrels)Barrel$() and fill it with water using a $(thing)Bucket$().$(br)For a full $(l:mechanics/barrels)Barrel$() of water you will need 20 $(l:mechanics/flux)Flux$().$(br)Place the $(l:mechanics/flux)Flux$() inside the $(l:mechanics/barrels)Barrel$() and seal it, as you do, the water should transform to limewater."),
```

Comment:
Don't say how much you need for a full barrel, say the exact ratio of flux to water you need